sdk/keyvault/Azure.Security.KeyVault.Administration/CHANGELOG.md

```diff
@@ -3,12 +3,15 @@
 ## 4.5.0-beta.1 (Unreleased)
 
 ### Features Added
+- Added support for service API version `7.6-preview.1`.
```

Comment:
Nit: you should put a space after headers. Install the markdown-lint extension for VSCode. It is required for learn.microsoft.com content and everyone should probably be using it.

Reply:
Same for all other markdown files herein. Keeping them markdown-lint clean helps avoids distracting blue squiggles when others with the extension (again, required for learn.microsoft.com content) make edits.